Index: y.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>console.log(1);\r\nconsole.log(2);
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/y.js b/y.js
--- a/y.js	(revision ade7241be882d4985f2c033300af24f93c23ef16)
+++ b/y.js	(date 1617816662399)
@@ -1,2 +1,2 @@
 console.log(1);
-console.log(2);
\ No newline at end of file
+console.log(2);
